common/common.go

```diff
@@ -185,3 +185,68 @@ func (l *PrivateNetList) All() bool {
 func (l *PrivateNetList) Specific(net string) bool {
 	return l.mapping[net]
 }
+
+type PrivateNetArgs struct {
+	mapping map[string]string
```

Comment:
Thanks for your input, I will look into this and post a follow-up PR for the both types after this one has been merged.